src/ui/flutter_app/lib/l10n/intl_hu.arb

```diff
@@ -55,35 +55,35 @@
   "@moveList": {
     "description": "Move list"
   },
-  "noGameRecord": "Nincs játékrögzítés",
+  "noGameRecord": "Nincs feljegyzés",
   "@noGameRecord": {
     "description": "No record"
   },
-  "ok": "Ok",
+  "ok": "Rendben",
   "@ok": {
     "description": "OK"
   },
-  "confirm": "Megerősítés",
+  "confirm": "Jóváhagy",
   "@confirm": {
     "description": "Confirm"
   },
-  "cancel": "Visszavonás",
+  "cancel": "Mégsem",
   "@cancel": {
     "description": "Cancel"
   },
-  "tipSelectWrong": "Helytelen korongot válasszon.",
+  "tipSelectWrong": "Válassza ki a rossz helyen lévő korongot!",
   "@tipSelectWrong": {
     "description": "Select the wrong piece."
   },
-  "tipPlace": "Rakjon le korongot.",
+  "tipPlace": "Rakjon le egy korongot!",
```

Comment:
I think it is more appropriate to use a period here. I will fix it later.

Reference: 
http://download.microsoft.com/download/e/6/1/e61266b2-d8b4-4fe0-a553-f01dc3976675/hun-hun-StyleGuide.pdf

Generally, we should follow the style of the English sentence. According to the Rules of Hungarian Grammar, exclamation mark must be used after each imperative sentence. However, we deviate from this rule in most cases. The reason behind it’s that UA and UI texts contain very high number of imperative sentences (instructions, procedures, etc.). It would look quite unnatural and could seem somewhat "aggressive" to place an exclamation mark after almost every sentence. Therefore, we follow the English style, and we place periods after simple imperative sentences:

Enter the file name. Adja meg a fájl nevét. Don’t show this message again. Ne jelenjen meg többé ez az üzenet.